README.md

```diff
@@ -11,25 +11,46 @@
 
 <!-- marker-introduction -->
 
-`metatrain` is a command line interface (CLI) to **train** and **evaluate** atomistic
-models of various architectures. It features a common `yaml` option inputs to configure
-training and evaluation. Trained models are exported as standalone files that can be
-used directly in various molecular dynamics (MD) engines (e.g. `LAMMPS`, `i-PI`, `ASE`
-...) using the [metatomic](https://docs.metatensor.org/metatomic) interface.
-
-The idea behind `metatrain` is to have a general hub that provides a homogeneous
-environment and user interface, transforming every ML architecture into an end-to-end
-model that can be connected to an MD engine. Any custom architecture compatible with
-[TorchScript](https://pytorch.org/docs/stable/jit.html) can be integrated into
-`metatrain`, gaining automatic access to a training and evaluation interface, as well as
-compatibility with various MD engines.
-
-> **Note**: `metatrain` does not provide mathematical functionalities *per se*, but
-> relies on external models that implement the various architectures.
+# What is metatrain?
+
+`metatrain` is a user-friendly command line tool that helps you **train machine learning
+models for atomistic systems**. In simple terms, it lets you create models that can
```

Comment:
Personally I find it distracting to have random **bold** bits *in a sentence*.. it IS a Bi*t* like having cRaZy capitalization, typically if I'm reading I'm reading the whole sentence, not soundbites